packages/solr.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+# The following script is pulling the binary files from the
+# recommended mirror site. Visit http://www.apache.org/dyn/closer.lua/lucene/solr/
+# for a full list of available mirrors.
+#
+# Remember to set the necessary version of Java. See
+# https://codeship.com/documentation/languages/java-and-jvm-based-languages/
+# for more information.
+#
+# To run this exact configuration, add the following to your builds:
+curl -sSL https://raw.githubusercontent.com/codeship/scripts/master/packages/solr.sh | bash -s
+
+# Set Java version
+jdk_switcher home oraclejdk8
+jdk_switcher use oraclejdk8
+
+# Install solr
+SOLR_VERSION="6.0.0"
+SOLR_MIRROR_URL="http://mirror.olnevhost.net/pub/apache/lucene/solr"
```

Comment:
`SOLR_VERSION` and the `SOLR_MIRROR_URL` should probably be editable. Right now the environment variables a user sets get replaced by the values on lin 19 & 20. See https://github.com/codeship/scripts/blob/master/packages/firefox.sh#L10-L11 for how to provide default values without replacing existing variables.